docs/source/reference/config.rst

```diff
@@ -244,6 +244,10 @@ Available fields and semantics:
     # instances. SkyPilot will auto-create and reuse a service account (IAM
     # role) for AWS instances.
     #
+    # NO_UPLOAD: No credentials will be uploaded to the pods. Useful for
+    # avoiding overriding any existing credentials that may be automounted on
+    # the cluster.
```

Comment:
Can we update the caveats below for `SERVICE_ACCOUNT` L268, i.e. if people need fully disable the upload, they should use `NO_UPLOAD`.